integration/ec2_test.go

```diff
@@ -282,3 +283,168 @@ func TestIAMNodeJoin(t *testing.T) {
 		return len(nodes) > 0
 	}, time.Minute, time.Second, "waiting for node to join cluster")
 }
+
+// TestEC2Labels is an integration test which asserts that Teleport correctly picks up
+// EC2 tags when running on an EC2 instance.
+func TestEC2Labels(t *testing.T) {
+	if os.Getenv("TELEPORT_TEST_EC2") == "" {
+		t.Skipf("Skipping TestLabels because TELEPORT_TEST_EC2 is not set")
```

Comment:
Is there any way to test this logic in such a way that you don't need to skip it? If experience tells me anything, we will basically. never set this var, the test will never run, and it won't be helpful to us.